package.json

```diff
@@ -52,6 +52,7 @@
     "rollup-plugin-typescript2": "0.34.0",
     "tslib": "2.4.0",
     "typescript": "4.8.3",
+    "unplugin-vue-components": "^0.22.7",
```

Comment:
nit: can you use an exact dependency here please?

Reply:
OK, I'm doing it